apps/root/src/state/create-position/reducer.ts

```diff
@@ -61,11 +64,13 @@ export default createReducer(initialState, (builder) => {
     .addCase(setFrequencyValue, (state, { payload }) => {
       state.frequencyValue = payload;
     })
-    .addCase(setFromYield, (state, { payload }) => {
-      state.fromYield = payload;
+    .addCase(setFromYield, (state, { payload: { option, manualUpdate } }) => {
+      state.fromYield = option;
+      state.userHasChangedYieldOption = manualUpdate && true;
     })
-    .addCase(setToYield, (state, { payload }) => {
-      state.toYield = payload;
+    .addCase(setToYield, (state, { payload: { option, manualUpdate } }) => {
+      state.toYield = option;
+      state.userHasChangedYieldOption = manualUpdate && true;
```

Comment:
Not sure why the && true is needed

Reply:
I want to update it to true only if manualUpdate is true, but as I'm writing this I realize it is not necessary hehe